orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/config/CloudDriverConfiguration.groovy

```diff
@@ -106,4 +109,10 @@ class CloudDriverConfiguration {
       .build()
       .create(KatoRestService)
   }
+
+  @Bean
+  ObjectMapper pipelineObjectMapper() {
+    new ObjectMapper()
+      .registerModule(new SimpleModule("MyConverterModule").addAbstractTypeMapping(Stage, PipelineStage))
```

Comment:
No need for `OrchestrationStage`? I suppose if we're doing an `instanceof Orchestration; return null` than it's unnecessary?